src/abstract.jl

```diff
@@ -206,23 +206,39 @@ An alias for size.
 shape(op::AbstractLinearOperator) = size(op)
 
 """
-    hermitian(op)
     ishermitian(op)
 
 Determine whether the operator is Hermitian.
 """
-hermitian(op::AbstractLinearOperator) = op.hermitian
 ishermitian(op::AbstractLinearOperator) = op.hermitian
 
 """
-    symmetric(op)
+    Hermitian(op, uplo=:U)
+"""
+function Hermitian(op::AbstractLinearOperator, uplo::Symbol = :U)
+  ishermitian(op) || throw(LinearOperatorException("Operator is not Hermitian"))
```

Comment:
Would it make sense for `Hermitian(op)` to return `(op + op') / 2)` when `op` itself is not Hermitian? That would be the Hermitian part of `op`.